### ISSUE TITLE
No module named 'tensorflow'. Anaconda+windows10+tensorflow-gpu+cuda8+cudnn6

### ISSUE THREAD

**Issue by notsmartenough** [tensorflow/tensorflow#16379]
I had anaconda on my windows 10.
I installed CUDA 8.0 with cuDNN 6 and then followed [http://blog.nitishmutha.com/tensorflow/2017/01/22/TensorFlow-with-gpu-for-windows.html](url) to activate tensorflow-gpu environment. Now when I import tensorflow in the console, it works but with jupyter notebook opened right in this environment it throws the error. I even upgraded setuptools as mentioned in a previous issue.
![capture5](https://user-images.githubusercontent.com/10391022/35360420-a5cd8f98-0183-11e8-919a-53da56df5ee8.JPG)
![capture6](https://user-images.githubusercontent.com/10391022/35360450-c27e44e8-0183-11e8-9c0a-5aaaa05000c4.JPG)
![capture7](https://user-images.githubusercontent.com/10391022/35360455-c7e24a42-0183-11e8-964b-7186fb233078.JPG)
![capture8](https://user-images.githubusercontent.com/10391022/35360549-174292c2-0184-11e8-9d6a-93ba6139a275.JPG)

**Comment by quaeler** [comment]
If, instead of just typing `jupyter` on command line from inside your active conda environment, you type `jupyter --paths` is it listing paths in the virtual environment?

**Comment by notsmartenough** [comment]
I don't think so.
I just started my computer and did `activate tensorflow-gpu`. Do I need to create a conda env?
![capture5](https://user-images.githubusercontent.com/10391022/35377497-6711e3f6-01d5-11e8-9688-d50af933a6cb.JPG)

**Comment by notsmartenough** [comment]
I realised jupyter is not installed in the tensorflow-gpu environment so I did `conda install jupyter` and now the `--path` shows the env. When I see`conda list` I realize many packages missing and everytime I have to install, I have to download, which consumes data. Is there a way to download once and install into every new environment I create? 
Btw,  now TF can be imported in jupyter, but still not in spyder. I guess I need to download and install spyder in this environment again? (I feel like there is a better solution)
